src/coreclr/jit/importer.cpp

```diff
@@ -13488,19 +13487,26 @@ void Compiler::impImportBlockCode(BasicBlock* block)
                 // Work is evidently required, add cast node
                 else
                 {
+                    GenTree* castOp = op1;
```

Comment:
I would just keep it as `op1`.

Reply:
Hmm, but I need to keep the pointer to the original node to check if it was actually constant after the casts have been introduced. Do you think it would be better to just check for the operand?

Reply:
Yeah, you could just do `op1->gtGetOp1()->OperIsConst()`.